sql/core/src/test/resources/tpcds-plan-stability/approved-plans-v1_4/q19.sf100/simplified.txt

```diff
@@ -13,12 +13,16 @@ TakeOrderedAndProject [ext_price,brand,brand_id,i_manufact_id,i_manufact]
                         InputAdapter
                           Exchange [ss_customer_sk] #2
                             WholeStageCodegen (4)
-                              Project [i_brand_id,i_brand,i_manufact_id,i_manufact,ss_customer_sk,ss_ext_sales_price,s_zip]
+                              Project [ss_customer_sk,ss_ext_sales_price,i_brand_id,i_brand,i_manufact_id,i_manufact,s_zip]
```

Comment:
Would this kind of thing possibly affect the order of columns from a `select *` or is that accounted for elsewhere?